<launch>

  <arg name="BAG_ID" />
  <arg name="SLEEP" default="10" />

  <param name="/use_sim_time" value="true" />

  <node name="rosbag_play_rs_rgbd"
        pkg="ros_objslampp_ycb_video" type="rosbag_play_rs_rgbd.py"
        clear_params="true"
        args="--id $(arg BAG_ID) --sleep $(arg SLEEP) --rosbag-args --clock"
        output="screen" launch-prefix="xterm -geometry 100x30 -e" required="true">
  </node>

  <group ns="camera">
    <node pkg="nodelet" type="nodelet" name="points_xyzrgb_aligned_depth_to_color"
          args="standalone depth_image_proc/point_cloud_xyzrgb">
      <remap from="rgb/image_rect_color"        to="color/image_rect_color" />
      <remap from="rgb/camera_info"             to="color/camera_info" />
      <remap from="depth_registered/image_rect" to="aligned_depth_to_color/image_raw" />
      <remap from="depth_registered/points"     to="depth_registered/points" />
    </node>
  </group>

</launch>
